src/pudl/transform/ferc1.py

```diff
@@ -2477,6 +2573,54 @@ def spot_fix_rows(self, df: pd.DataFrame) -> pd.DataFrame:
         return df
 
 
+class ElectricEnergyAccountSourcesTransformer(Ferc1AbstractTableTransformer):
+    """Transformer class for :ref:`electric_energy_account_sources_ferc1` table.
+
+    For XBRL, this is a duration-only table. Right now we are merging in the metadata
+    but not actually keeping anything from it. We are also not yet doing anything with
+    the sign.
+    """
+
+    table_id: Ferc1TableId = Ferc1TableId.ELECTRIC_ENERGY_ACCOUNT_SOURCES_FERC1
+    has_unique_record_ids: bool = False
+
+    def normalize_metadata_xbrl(
+        self, xbrl_fact_names: list[str] | None
+    ) -> pd.DataFrame:
+        """Normie.
+
+        Determine whether the
```

Comment:
Docstring?

Reply:
Nahhh. jk yeah i'll flesh this out... this was one where probably more things need to be moved into the generic method but i couldn't quite tell how much yet.